pureconfig/README.md

```diff
@@ -0,0 +1,17 @@
+# Toggle
```

Comment:
This documentation needs to go here https://github.com/avast/scala-server-toolkit/blob/master/site/docs/subprojects/pureconfig.md so that it's part of the generated docs site.